README.md

```diff
@@ -1 +1,70 @@
-# the-book-store
\ No newline at end of file
+# Getting Started with Create React App
```

Comment:
- [ ] Kindly update your README file to be more descriptive of this project. I would recommend using this [Microverse template](https://github.com/microverseinc/readme-template)
- [ ] Your project is missing the remaining redux files setup according to the image below. There should be a `books` folder with a `books.js` file. There should also be a `categories` folder with a `categories.js` file. You are also missing the `configureStore.js` file. Kindly add these files.
  
  ![grace-file](https://user-images.githubusercontent.com/36542986/146185703-b3f1e733-10f8-47c5-ba22-7b85e749fa0f.jpg)

Reply:
done